canonical/canonical.com
# WD-13772 Refresh /data/postgresql

## Done

- Refreshed /data/postgresql
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- [Design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=834-6108&node-type=FRAME&t=wNiWN8kj6583ohSx-0)
- [Copydoc](https://docs.google.com/document/d/1GxineF4WLt0YlShaYkrzqWjIOkXjLJEfsnUdWVfWnz4/edit)
- [Demo](https://canonical-com-1362.demos.haus/data/postgresql)

## Issue / Card

Fixes # [WD-13772](https://warthogs.atlassian.net/browse/WD-13772)



[WD-13772]: https://warthogs.atlassian.net/browse/WD-13772?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
